pyews/ews/request_response.py

```diff
@@ -253,6 +253,18 @@ def set_itemids (self, index, node):
         item.itemid.set(itemid)
         item.change_key.set(ck)
 
+    def _get_items(self):
+        "Returns items provided to in the request to create remote records"
```

Comment:
(a) Always use a space after method name in the definition.

(b) Always use Triple Quotes for doc strings (https://www.python.org/dev/peps/pep-0257/)